enzymaticReactions.py

```diff
@@ -13,3 +14,6 @@
 """
 Interim commit #3
 """
+
+for i in range(0,5):
```

Comment:
wow! are you showing all the reactions we have to model? using this `for i in range(0,5):` command?

Reply:
Yes... what does it look like im showing.... I'm an **ML** god.